README.md

```diff
@@ -408,7 +408,7 @@ const handleSaveCalculation = ({key}) => {
         'default': throw Error('Unknown action')
     }
     
-    (actions[key] || 'default')()
```

Comment:
I believe there were a few issues here:
1. Can you call this expression as a function? Something else might be the intention here 🤔
2. It relies on `actions[key]` to return a falsy value to evaluate the right part of the `||`, but that falsy value might actually be OK or expected.
3. The default value is just always `default`
4. Nothing is returned? :)